core/src/test/scala/unit/kafka/log/LogCleanerTest.scala

```diff
@@ -1086,7 +1086,7 @@ class LogCleanerTest extends JUnitSuite {
     val end = 2
     val offsetSeq = Seq(0L, 7206178L)
     writeToLog(log, (start until end) zip (start until end), offsetSeq)
-    cleaner.buildOffsetMap(log, start, end, map, new CleanerStats())
+    cleaner.buildOffsetMap(log, start, 7206178L + 1L, map, new CleanerStats())
```

Comment:
Perhaps we can choose better names for start and end to avoid the confusion. Maybe `keyStart` and `keyEnd` or something like that?

Reply:
Done